src/factories/icon.ts

```diff
@@ -4,11 +4,13 @@ import { waitForIconCull } from '@/objects/culling'
 import { getIconTexture } from '@/textures/icon'
 
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export async function iconFactory() {
+export async function iconFactory(disableCulling?: boolean) {
```

Comment:
Can you make this an object so we aren't passing plain booleans? Reads better as `iconFactory({ cull: false })` or something like that. Same for label factory.